apps/web/src/app/api/v1/events/route.ts

```diff
@@ -0,0 +1,41 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { findAllEvents } from '@hypha-platform/core/server';
+import { db } from '@hypha-platform/storage-postgres';
+import { EVENT_ENTITY_TYPES, EventEntity } from '@hypha-platform/core/client';
+
+export async function GET(request: NextRequest) {
+  const { searchParams } = new URL(request.url);
```

Comment:
[Nextjs already has an URL object inside](https://nextjs.org/docs/15/app/api-reference/functions/next-request#nexturl) so no need to create a new one
```suggestion
  const { searchParams } = request.NextUrl;
```